test/integration/targets/vyos_lldp_interfaces/tests/cli/deleted.yaml

```diff
@@ -0,0 +1,44 @@
+---
+- debug:
```

Comment:
For the idempotent tests, please assert if length of command list is also 0.

Reply:
Done